aboutlibraries/src/main/java/com/mikepenz/aboutlibraries/LibsFragmentCompat.kt

```diff
@@ -84,6 +86,28 @@ class LibsFragmentCompat {
 
         recyclerView.doOnApplySystemWindowInsets(Gravity.BOTTOM, Gravity.START, Gravity.END)
 
+        itemAdapter.itemFilter.filterPredicate = fun (item, constraint): Boolean {
```

Comment:
Could potentially expose `filterPredicate` as a configurable option in the future, but for now I think searching by `libraryName` should suffice

Reply:
Was actually going to propose to expand this also for the identifier so we would cover the `maven coordinate` when searching, in case the lib name does not match.